### PR TITLE
[issue 1369] idempotent kernel modules

### PR DESCRIPTION
**Pull request by shawndwells**
resolves https://github.com/OpenSCAP/scap-security-guide/issues/1369

### CODE REVIEW

File: shared/templates/template_BASH_kernel_module_disabled [hunk 1]
```diff
@@ -4,4 +4,9 @@
 # complexity = low
 # disruption = medium
 # THIS FILE IS GENERATED. DO NOT EDIT!
```
Comment (mpreisler) [inline shared/templates/template_BASH_kernel_module_disabled]: saw this comment?

Comment (shawndwells) [inline shared/templates/template_BASH_kernel_module_disabled]: Those reminders were put there in https://github.com/OpenSCAP/scap-security-guide/commit/ae24612b1569d8c7c65630db892505541b29f7e0. Reminder to users the remediation files are auto-generated.

Comment (mpreisler) [inline shared/templates/template_BASH_kernel_module_disabled]: I am really confused now. The comment says you shouldn't edit the file directly. Yet this PR edits the file directly and doesn't edit any template or other file. So it seems it's done contrary to the warning in the generated file. Am I missing something?

Comment (shawndwells) [inline shared/templates/template_BASH_kernel_module_disabled]: This PR edits the template to make it idempotent.

Comment (mpreisler) [inline shared/templates/template_BASH_kernel_module_disabled]: Ah, lol. I am slow today I guess.

**Comment by mpreisler** [comment]
ACK

**Pull request merged**